zedd-app/src/AppState.ts

```diff
@@ -297,6 +297,9 @@ export class AppState {
   @observable
   public changingSliceTask: TimeSlice | undefined = undefined
 
+  @observable
+  public clarityViewFilterProject: string
```

Comment:
This value is specific to the clarity view. It only relevant for rendering. Therefore it makes sense to keep this state inside the ClarityView component using React.useState